README.md

```diff
@@ -397,6 +397,102 @@ Here `"subject":"005xx0000012345"` is the salesforce id of sender.
     }
   }
 }
+```  
+
+#### ConversationHistory API
+
+The API supports validation to ensure that partners don’t import conversations older than 24 hrs.
+
+Example request for sending conversation History to MCS, be sure to replace placeholder values `<..>`
+
+```bash
+curl -v \
+-H "Authorization: Bearer <AccessToken>" \
+-H "content-type: application/json" \
+-H "OrgId: <OrgId>" \
+-H "AuthorizationContext: <AuthorizationContext>" \
+-H "AuthorizationContextType: CONVERSATIONCHANNELDEFINITION" \
+-H "RequestId: <RequestId>" \
+-X POST -d '{
+  "channelAddressIdentifier": "<channelAddressIdentifier>",
+  "conversationParticipants": [
```

Comment:
why do we have a different variants of example. Other APIs are added with JSON encoded format but for this API, we have added JSON payload. Can you please ensure it's consistent to other APIs?

Reply:
updated.